.prettierignore

```diff
@@ -1,2 +1,3 @@
 .next
-.contentlayer
\ No newline at end of file
+.contentlayer
+**/*.mdx
```

Comment:
Preventing Prettier from formatting Markdown fixes the issue with multiline JSX  causing weird rendering issues (e.g. The `<Callout />` component).

I'm 50/50 on this. I think it's nice having a formatter for Markdown files. Perhaps we revert this and just make sure we save without formatting for files with JSX?